lib/tasks/adhoc.rake

```diff
@@ -0,0 +1,14 @@
+namespace :adhoc do
+  desc 'Extract a list of All articles and what categories they sit in'
+  task :extract_page => :environment do
+    File.open('pages_with_cat.csv','w') do |file|
+      file.write "Url\tCategories\n"
+      Comfy::Cms::Page.where(state: :published).find_each do |page|
+        file.write "/en/articles/#{page.slug}\t#{page.categories.map(&:label).join(', ')}\n"
```

Comment:
Agree with @jongilbraith. Probably you want: 

```ruby
file.write "/#{page.site.locale}/articles/#{page.slug}\t#{page.categories.map(&:label).join(', ')}\n"
```

Explaining: You can do `english  = Comfy::Cms::Site.find_by(label: 'en')`or `welsh = Comfy::Cms::Site.find_by(label: 'cy')`
And then you can access using `english.pages` or `welsh.pages`.
The site object has *locale* or *label* as attribute to get the right locale. You choose.